_posts/2018-05-15-2017-recap.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title:  "2017 Reflections &#124; Projections: A Recap"
+date:   2018-05-15 12:00:00
+image: "/assets/images/squiggles2.png"
+categories: spotlight
+tags: rp2018
+author: { 
+  name: Rohin Bhasin, 
+  bio: "CS @ Illinois ’21",
+  title: "R&#124;P 2018 Marketing Staff",
+  image: "/assets/images/logo.png" #default
```

Comment:
can you link your pic (it should be `"/assets/images/author_images/rohin_bhasin.png"`)